qjcr.yaml

```diff
@@ -2,7 +2,7 @@
 qjcr:
   id: qjcr
   name: Queens' College JCR
-  baseUrl: "/"
+  baseUrl: "/qjcr."
```

Comment:
should end in slash not dot?

Reply:
Good catch - fixed https://github.com/camvote/bob-providers/pull/3/commits/d4117e34b27551f473f4be0f4fda1b0e6fa31a3d